11/umbraco-cms/reference/notifications/hot-vs-cold-restarts.md

```diff
@@ -6,7 +6,7 @@ description: When rebooting an Umbraco CMS website it is common to distinguish b
 
 The load time of your site is dependent on a few different things. When talking about hot vs. cold restarts of your Umbraco CMS website it comes down to whether NuCache needs to be rebuilt.
 
-* **Cold restart**: When ["In Memory Auto"](../templating/modelsbuilder/builder-modes.md#in-memory) mode is enabled and the NuCache needs to rebuild.
+* **Cold restart**: When NuCache needs to rebuild.
```

Comment:
Since you're removing the link here - Does it make sense to add it to the "In Memory Auto" section you've added further down? 🤔

Reply:
That's a very good point, I've re-added the link to the In Memory Auto section